packages/web-commons/src/charts/generators.ts

```diff
@@ -42,8 +42,8 @@ const generateChartData = async (props: {
       case ChartType.Trend:
         result[chartId] = generateTrendChart(chartOption, storeData, reportName)!;
         break;
-      case ChartType.Pie:
-        result[chartId] = generatePieChart(chartOption, storeData)!;
+      case ChartType.CurrentStatus:
```

Comment:
Ideally if user will have choice:
- use pre-defined chart, like current status pie chart
- use his-own pie chart configuration
At this moment, it's ok to have pre-defined charts only, but in future we should provide a fully customizable dashboard experience